test_autoarray/fit/plot/test_fit_imaging_plotters.py

```diff
@@ -27,27 +27,48 @@ def test__visuals_in_constructor_use_imaging_and_include(fit_imaging_7x7):
     )
 
     assert fit_imaging_plotter.visuals_2d.origin == (1.0, 1.0)
-    assert fit_imaging_plotter.visuals_with_include_2d.origin == (1.0, 1.0)
+    assert fit_imaging_plotter.get_2d.via_fit_imaging_from(

```

Comment:
Create fit_image_plotter in a fixture and separate out groups of assertions into different tests

Reply:
I actually meant to delete this unit test anyway.